packages/cdktf/lib/terraform-backend.ts

```diff
@@ -27,4 +27,13 @@ export abstract class TerraformBackend extends TerraformElement {
       },
     };
   }
+
+  public toMetadata(): any {
+    return {
+      backend: this.name,
+      ...(Object.keys(this.rawOverrides)
+        ? { overrides: { backend: this.rawOverrides } }
```

Comment:
As this is not using `Object.keys()` here, couldn't it leak the values of the overrides? Is that a mistake or on purpose?

Reply:
Ah forgot it here :)